src/main/java/org/molgenis/vcf/decisiontree/runner/info/NestedInfoSelector.java

```diff
@@ -0,0 +1,8 @@
+package org.molgenis.vcf.decisiontree.runner.info;
+
+import org.molgenis.vcf.decisiontree.filter.Allele;
+
+public interface NestedInfoSelector {
+
+  boolean isMatch(String infoValue, Allele allele, NestedInfoHeaderLine nestedInfoHeaderLine);
```

Comment:
- How about `String select(List<String> infoValues, ...` such that the class is response for returning one selection in case of multiple mathing hits.
- How about passing a parsed infoValue object that already has the correct types.